annotationProcessor/build.gradle

```diff
@@ -31,10 +36,16 @@ repositories {
 }
 
 dependencies {
+    // added to address synk warnings
+    constraints {
+        implementation 'com.azure:azure-identity:1.12.2'
```

Comment:
Is this due to `terra-common-lib`, and if so, is it possible to exclude it instead of pegging the version here?

Reply:
Yes, it is from common-lib. exclusion is a better option, done.